src/js/vre/chapter36/config/form.js

```diff
@@ -52,6 +52,14 @@ const requiredDateRange = _.merge(dateRange, {
   required: ['to', 'from']
 });
 
+export function validateBooleanGroup(errors, userGroup, form, schema, errorMessages = {}) {
```

Comment:
This is a new validation, should this be included in these changes or do we not want to restrict the options in this way, or should we table this for now and revisit it later?

Reply:
Could be a two stage question. Have you applied for benefits and then disclose the additional options if they select yes.

Reply:
I think this validation makes sense. We should run it by @stacyhlinn, but I think we can keep it unless we change the question format, like Rian suggests.

Reply:
However, the name clashes with a similar validation function we have elsewhere, so maybe `validateNoneOrSomeChosen` or something like that?